app/scripts/onlyKey/OnlyKeyComm.js

```diff
@@ -2040,7 +2040,8 @@ function saveBackupFile(e) {
   var backupData = ui.backupForm.backupData.value.trim();
   if (backupData) {
     d = new Date();
-    var filename = "onlykey-backup-" + d.toISOString() + ".txt";
+    Date.prototype.__proto__.format = require("es5-ext/date/#/format");
```

Comment:
Found this existing date format module in the source tree which saves doing all the formatting by hand. Not sure if slapping it on the Date class in this function is exactly standard JavaScript however...

Reply:
Right -- it is not a best practice to modify native JS prototypes. I understand the intent here, and we'll update the filenames in a future release. Thank you!